config/install/media_entity.bundle.twitter.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media_entity_twitter
+id: twitter
+label: Twitter
```

Comment:
All other bundles use name of the media type in singular form, which is "Tweet" in this case. I think that we should be consistent.